### PR TITLE
feat(web): restyles prediction banner

### LINKED ISSUE #2221: [iOS] Suggested tweaks to iOS UX from FPCC meeting
* Language name does not appear on space bar
* Make suggested predictions gray (not buttons) - styling

### LINKED ISSUE #2517: Word suggestion bar color match with iOS
**Is your feature request related to a problem? Please describe.**
Please looking at the iOS default keyboard, word suggestion bar. 
Suggestion buttons are overwhelming compared to the keys.
iOS system keyboard doesn't use buttons. 

**Describe the solution you'd like**
1. There is no need of white boxes
2. Need to vertical bars to between three choices similar to iOS keyboard
IMG_3721
IMG_3720

**Product context**
 - OS: iOS 13.3

**Additional context**
I assume this is a simple change but it will make Keyman keyboard native to iOS.

### LINKED ISSUE #2522: First word suggestion from the dictionary is not displaying quotes correctly on RTL text.
I'm using the dictionary (word list) feature for a right-to-left language (Balochi). The keyboard is called Balochi phonetic. When I type a letter it shows the leftmost word as being in quotes. The problem is there seems to be a paragraph flow issue so that that one letter is on the left and both the opening and closing quotes are on the right. See screenshot.
Screenshot 2020-01-23 at 10 42 05 AM

### PR DESCRIPTION
**Pull request by jahorton**
Fixes #2221.  (The first half has already been addressed.)
Fixes #2517.

This PR restyles the predictive banner to be more similar to the native iOS banner, rather than rendering the suggestions on top of "keys".

Here's some screenshots of the new looks:

Screen Shot 2020-02-11 at 2 19 41 PM
Screen Shot 2020-02-11 at 2 19 55 PM
Screen Shot 2020-02-11 at 2 22 26 PM
Screen Shot 2020-02-11 at 2 23 05 PM
Screen Shot 2020-02-11 at 3 11 12 PM
Screen Shot 2020-02-11 at 3 02 11 PM

Note that the CSS change for `overflow-x` still allows scrolling; it simply keeps the scrollbar hidden.  Do feel free to test this, of course.

----

As a side note, there are a few details possibly worth noting in the images above - for the iOS tablet images, note how the background color of the keyboards don't match.  Some investigation for Android colors may also be merited.  Of course, these details can be addressed in a different PR too.

### CODE REVIEW

**Comment by mcdurdin** [comment]
Looking good! Can you move the suggestion text down a few pixels so it's centered in the gray area, on iOS at least?

**Comment by jahorton** [comment]
Alright.  A bit of investigation later, I noticed something interesting in the existing CSS when used in the Android and iOS apps:

Screen Shot 2020-02-13 at 9 53 30 AM

Screen Shot 2020-02-13 at 9 54 51 AM

Note how when embedded within Keyman for iOS, the `6px` top gap disappears.  The easy solution?  When within the app, just increase the top gap by an extra `4px`.

So, with the increase:

Screen Shot 2020-02-13 at 10 23 30 AM
Screen Shot 2020-02-13 at 10 24 08 AM
Screen Shot 2020-02-13 at 10 33 33 AM
Screen Shot 2020-02-13 at 10 25 36 AM
Screen Shot 2020-02-13 at 10 26 05 AM

That said, it looks like a bit more of a boost would be merited on the largest resolution iPads.  It's like something is throwing off the positioning of the banner bar itself when embedded within the iPhone app, but narrowing down what that is might be problematic.

**Comment by mcdurdin** [comment]
That slight tweak does wonders! Looks so good. I can see a possibility for a future feature: control number of suggestions to show -- we've got lots of space on some devices and it may be helpful

File: web/source/resources/osk/kmwosk.css [hunk 1]
```diff
@@ -272,8 +272,9 @@
 .kmw-title-bar-actions{position: absolute; cursor: default; right:2px; top:0; width: 33px; height: 13px;}
 .kmw-footer-caption{color:#fff;font:0.7em Arial;margin:0 0 0 4px;}
 
-.kmw-banner-bar{height:100%; width:100%; margin:0; background-color:darkorange; display: inline-block;}
-.kmw-banner-bar .kmw-suggest-option {background-color: orange; display:inline-block; text-align: center; height: 85%; border-radius: 5px;overflow-x:scroll}
+.kmw-banner-bar{height:100%; width:100%; margin:0; background-color:darkorange; display: inline-block; white-space: nowrap;}
```
Comment (mcdurdin) [inline web/source/resources/osk/kmwosk.css]: When we update the CSS, can we reformat it to multi-line and indented? There's no need for it to all be one-line like this (I would suggest a future update would be to use SASS so we can reduce the complexity immensely but that's not for today).

**Pull request merged**